### PR TITLE
Migrate Roles screen to Angular

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6374

**Description**

  - Add new methods in RoleService
  - Create an Angular component for the new OrgSettingsRoles screen
  
**Additional context**

Before:
image

After:
image

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-tgmnmdckzb.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6374-migrate-role-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/scss/gio-global-style.scss [hunk 1]
```diff
@@ -49,3 +49,8 @@ $typography: map.get(theme.$gio-theme, typography);
 .mat-hint {
   font-size: mat.font-size(theme.$gio-typography, caption);
 }
+
+// Remove card padding when a mat list is inside a card
+.mat-card > .mat-list {
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/scss/gio-global-style.scss]: maybe we could add this like this : 
https://github.com/gravitee-io/gravitee-ui-particles/tree/main/ui-particles-angular/projects/ui-particles-angular/src/scss/mat-override
with story to illustrate override 

WDYS ?

File: gravitee-apim-console-webui/src/organization/configuration/tenants/org-settings-tenants.component.html [hunk 2]
```diff
@@ -28,7 +28,7 @@ <h4>
 </div>
 
 <div class="mat-elevation-z3">
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/tenants/org-settings-tenants.component.html:30]: 📝 i remove this in one of my PR. and add this by default in gio table wrapper

File: gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-roles.component.scss [hunk 3]
```diff
@@ -1,5 +1,59 @@
 @use '../../../scss/gio-layout' as gio-layout;
 
+@use 'sass:map';
+@use 'node_modules/@angular/material' as mat;
+@use '../../../scss/gio-material-theme' as gio;
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/organization/configuration/roles/org-settings-roles.component.scss]: :tavenier-conflict-run-away:

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/scss/gio-global-style.scss (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/scss/gio-global-style.scss]: Done in https://github.com/gravitee-io/gravitee-ui-particles/pull/9

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-espngvipem.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6374-migrate-role-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**